entries/l/liveoakbank.com.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Live Oak Bank": {
+      "domain": "liveoakbank.com",
+      "img": "liveoakbank.com.svg",
+      "tfa": [
+        "sms",
+        "call",
+        "totp",
+        "custom-software"
+      ],
```

Comment:
I see that you haven't added any documentation link. Could you find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach screenshots of the 2FA setup and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment like this:
![screenshot](https://user-images.githubusercontent.com/3535780/187552643-08eea7e8-e6c6-4873-b31c-0faf4347f69f.gif)